src/Avalonia.OpenGL/GlInterfaceBase.cs

```diff
@@ -23,7 +23,7 @@ public class GlInterfaceBase<TContext>
         public GlInterfaceBase(Func<string, IntPtr> getProcAddress, TContext context)
         {
             _getProcAddress = getProcAddress;
-            foreach (var prop in this.GetType().GetProperties())
+            foreach (var prop in this.GetType().GetProperties(BindingFlags.NonPublic | BindingFlags.Public | BindingFlags.Instance | BindingFlags.Static))
```

Comment:
Initializing static properties from an instance constructor is usually a bad idea.

Reply:
There is other way to access to private members ?

Reply:
Yes, but not including `BindingFlags.Static`. 
See https://docs.microsoft.com/en-us/dotnet/api/system.type.getproperties?view=netcore-3.1#System_Type_GetProperties_System_Reflection_BindingFlags_ and https://docs.microsoft.com/en-us/dotnet/api/system.reflection.bindingflags?view=netcore-3.1

Reply:
You right, thanks for clarification.